Documentation updates (#407)
cap-js/docs: guides/databases-postgres.md

1. Install and run [Docker Desktop](https://www.docker.com)

> 2. Create a file called `pg.yml` in your project root directory:

Reviewer: ```suggestion
2. Create the following file in your project root directory:
```
